abi-loader.ts

```diff
@@ -0,0 +1,47 @@
+import {ContractRegistryKey} from "./test/driver";
+import {ContractName} from "./typings/contracts";
+
+export function getAbiByContractRegistryKey(key: ContractRegistryKey): object {
+    switch (key) {
+        case 'protocol':
```

Comment:
consider using a global constant declaration for each contract key name instead of inline strings in several locations.

Reply:
Right

Reply:
On the other hand, we have the ContractRegistryKey type which I think is close enough. So I leave this for a separate PR.